### PR TITLE
Add nftables configuration

### PR DESCRIPTION
**Pull request by yoe**
The portmapper service can be used in a DDoS amplification attack. Don't
allow our servers to be misused.

(note: firewalls have been installed currently, but outside of ansible)

### CODE REVIEW

File: ansible/playbooks/roles/encoder-network/files/nftables.conf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+table inet filter {
```
Comment (SuperQ) [inline ansible/playbooks/roles/encoder-network/files/nftables.conf]: Let's keep some of the defaults from the Debian included config here.

```suggestion
#!/usr/sbin/nft -f
{{ ansible_managed | comment }}

flush ruleset

table inet filter {
```

File: ansible/playbooks/roles/encoder-network/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+- name: Ensure nftables is installed
+  apt:
+    package: nftables
+    state: latest
+
+- name: Create nftables config file
+  copy:
```
Comment (SuperQ) [inline ansible/playbooks/roles/encoder-network/tasks/main.yaml]: Let's make this a template, so we can make it more flexible.

```suggestion
  template:
```

File: ansible/playbooks/roles/encoder-network/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    src: nftables.conf
+    dest: /etc/nftables.conf
+
+- name: Activate nftables config
+  service:
+    name: nftables
+    state: restarted
```
Comment (SuperQ) [inline ansible/playbooks/roles/encoder-network/tasks/main.yaml]: This is not idempotent, move this to `handlers/main.yml` and use a `notify` on the config file.

**Comment by paulfantom** [comment]
Or you could use on of https://wiki.nftables.org/wiki-nftables/index.php/Using_configuration_management_systems#ansible. Especially [Frzk/ansible-role-nftables](https://github.com/Frzk/ansible-role-nftables) looks quite good.

But maybe this is for FOSDEM 2021 :)

**Comment by yoe** [comment]
> Or you could use on of https://wiki.nftables.org/wiki-nftables/index.php/Using_configuration_management_systems#ansible. Especially [Frzk/ansible-role-nftables](https://github.com/Frzk/ansible-role-nftables) looks quite good.
> 
> But maybe this is for FOSDEM 2021 :)

Yeah, I definitely think it's way too late in the process to go that route now.

I was tempted to say the same about the template (unlikely we're going to add more to it for now), but meh, I guess it doesn't hurt, and having the "ansible managed" comment could definitely help. Then again, we can also hardcode that.

**Pull request merged**